cypress/e2e/facility_spec/facility_manage.cy.ts

```diff
@@ -24,6 +24,10 @@ describe("Facility Manage Functions", () => {
   const currentOccupied = "80";
   const totalUpdatedCapacity = "120";
   const currentUpdatedOccupied = "100";
+  const totalOxygenCapacity = "100";
```

Comment:
delete the existing unused constants or reuse them when you are modifying the existing test

Reply:
new ones are added for oxygen and earlier ones are for doctor and bed